dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -345,6 +342,12 @@
             name="%excludeAllHandledTypesClasses.name"
             description="%excludeAllHandledTypesClasses.desc"
             required="false" type="Boolean" />
+            
+        <!--com.ibm.ws.webcontainer.-->
```

Comment:
This needs to be beta guarded. We should open an issue to remove the beta guard when ready.